lib/security/JweToken.ts

```diff
@@ -74,6 +118,84 @@ export default class JweToken extends JoseToken {
     return Buffer.from(jweString);
   }
 
+  /**
+   * Encrypts the given string in JWE JSON serialized format using the given key in JWK JSON object format.
```

Comment:
Unclear what the "given string" is. Is it `this.content`? If so, it is not given to this method as an argument as one would conventionally think.